lib/RTCSession.js

```diff
@@ -3523,4 +3541,17 @@ module.exports = class RTCSession extends EventEmitter
       video
     });
   }
+
+  _getDTMFRTPSender()
+  {
+    const dtmfSender = this._connection.getSenders().find((sender) =>
+    {
+      return sender.track && sender.track.kind === 'audio';
+    });
+
+    if (dtmfSender)
```

Comment:
Is must also be checked whether the sender can send DTMFs via [canInsertDTMF](https://www.w3.org/TR/webrtc/#dom-rtcdtmfsender-caninsertdtmf) method

``` javascript
if (dtmfSender && dtmfSender.dtmf && dtmfSender.dtmf.canInsertDTMF)
{
 return dtmfSender.dtmf;
}